test/generator/gpu_object_lifetime_aottest.cpp

```diff
@@ -162,7 +162,11 @@ int main(int argc, char **argv) {
             {
                 Buffer<int> copy(raw_buf);
             }
-            halide_device_free(nullptr, &raw_buf);
+            int result = halide_device_free(nullptr, &raw_buf);
```

Comment:
Failure should be impossible in this specific context again. The point of this test is to create a trace of debug events so that we can check for lifetime issues. I guess this is a secondary test of if the buffer becomes malformed or something.

Reply:
Interesting that the compiler only flagged *some* of these with unused result :-/

Reply:
Done